cpp/tests/pagerank/mg_pagerank_test.cpp

```diff
@@ -316,7 +311,22 @@ TEST_P(Tests_MGPageRank_File, CheckInt32Int32FloatFloat)
 TEST_P(Tests_MGPageRank_Rmat, CheckInt32Int32FloatFloat)
 {
   auto param = GetParam();
-  run_current_test<int32_t, int32_t, float, float>(std::get<0>(param), std::get<1>(param));
+  run_current_test<int32_t, int32_t, float, float>(
+    std::get<0>(param), override_Rmat_Usecase_with_cmd_line_arguments(std::get<1>(param)));
```

Comment:
A downside to this approach is that gtest doesn't know the override is happening, and as a result it may end up producing redundant combinations of input params (since each param is being overridden the same way), if there are >1 rmat usecase instances in the param list.  I'm guessing `--gtest_list_tests` may not print the correct list of tests when one of the rmat options are passed either (eg. `TestSuite/TestCase/2` will be associated with the default params and not the overridden ones), but that's relatively minor.

Unfortunately I don't have a better suggestion that's in the scope of this PR though.

Reply:
Yeah... maybe it might be better to rename ```rmat_larget_tests``` to something like ```rmat_benchmarks``` (@ChuckHastings  also mentioned about creating Benchmark_MGPageRank_Rmat) and enforce rmat_benchmarks to have only one Rmat_Usecase (and in the benchmark mode, just select rmat_benchmarks using --gtest_filter).

Reply:
Yeah... maybe it might be better to rename rmat_larget_tests to something like rmat_benchmarks (@ChuckHastings also mentioned about creating Benchmark_MGPageRank_Rmat) and enforce rmat_benchmarks to have only one Rmat_Usecase (and in the benchmark mode, just select rmat_benchmarks using --gtest_filter).